terminal/TerminalClient.cpp

```diff
@@ -165,6 +168,39 @@ vector<pair<int, int>> parseRangesToPairs(const string& input) {
 }
 
 int main(int argc, char** argv) {
+  // easylogging parse verbose arguments, see [Application Arguments]
+  // in https://github.com/muflihun/easyloggingpp/blob/master/README.md
+  START_EASYLOGGINGPP(argc, argv);
```

Comment:
Let's create a LogHandler.h/.cpp and move this macro, the INITIALIZE_EASYLOGGINGPP macro, and the config below into those files.